files/build_templates/sonic_debian_extension.j2

```diff
@@ -138,11 +138,11 @@ sudo chmod a+x $FILESYSTEM_ROOT/usr/sbin/policy-rc.d
 
 {% if installer_debs.strip() -%}
 {% for deb in installer_debs.strip().split(' ') -%}
-if [ $sonic_asic_platform == "mellanox" ]; then
+{% if sonic_asic_platform == "mellanox" or sonic_asic_platform == "cavium" %}
```

Comment:
use dh_installinit to automatically generate the post-install script, it will use the return code of policy-rc.d to determine whether to run the post-install script or not.